### ISSUE TITLE
How to jump into editing metadata manually ?

### ISSUE THREAD

**Issue by aahnik** [deepjyoti30/ytmdl-web-v2#23]
# Question

## Checklist

- [x] I've looked through the README and the WiKi for something similar.
- [x] I tried talking with the developer on the Telegram support group.

## Question

First I enter the youtube URL, and then the page comes asking for name of song. But I am actually downloading an audiobook, and I don't want to search for metadata, I want to straight go into writing metadata on my own.

In the metadata search page, I gave some name, but it loads forever.

**Comment by deepjyoti30** [comment]
@aahnik As of now there's no option to directly go to that page but I think adding a setting for that won't be a bad idea. 

We can have a setting that will automatically skip the search meta page and go to manual meta.

**Comment by deepjyoti30** [comment]
@aahnik Added a setting to enable jumping to manual mode directly from the search page. It will be available once the deployment is done.

You can raise a new issue if you find any issue with the feature!
